.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,19 +35,19 @@ Using the PR template:
 
 ### Contributor Checklist
 
-- [ ] PR has the correct target milestone
+- [ ] PR has the correct target target branch and milestone
 - [ ] PR has 'needs review' or 'work-in-progress' label
-- [ ] If PR is ready for review, it has been assigned or requests review from someone and labeled as 'needs review'
-- [ ] PR has been fully tested manually
-- [ ] Documentation is updated
-- [ ] External dependencies files were updated (`yarn` and `pip`)
-- [ ] Link to diff of internal dependency change is included
+- [ ] Contributor has fully tested the PR manually
```

Comment:
Maybe just "PR was tested manually" ?

Reply:
I wanted to include it for both contributor and reviewer. too much?

Reply:
No matter how small a change: The contributor should spin things up manually to see how they behave.

If I'm reviewing and see that the contributor states they have tested something manually, and it's a small change.. I would like to be able to skip the reviewer's test. I think we can manage within the circle of core reviewers to manage a balance of when to test manually? So could perhaps remove from reviewer's tasks :)

Reply:
Yeah, that's up to the reviewer's discretion. [1] How about I update the guidance at the top from:

> 3. Skip checklist items that aren't applicable

to

>  3. Skip checklist items that are not applicable or not necessary

then if the reviewer (or contributor) decide to skip testing, it's documented.

----

[1] Though it does seem risky. At a high level, I think 'trust but verify' is much safer than just 'trust' because we often catch issues like unhandled errors, linting issues, and broken tests even for trivially small changes.